package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@fabrictech/dynamodb-localhost",
-  "version": "0.3.2",
+  "version": "0.3.3",
   "engines": {
-    "node": "^10.18.0",
+    "node": ">10.18.0",
```

Comment:
We can set this to ^12.22.1 (the intended node upgrade version) later

Reply:
NBD here, but I believe you can specify multiple supported versions as well, with either [a `||`](https://stackoverflow.com/a/53071208). Or, we could define a [range](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines)